Jellyfin.Api/Controllers/SubtitleController.cs

```diff
@@ -346,5 +353,95 @@ public async Task<ActionResult> GetRemoteSubtitles([FromRoute, Required] string?
                 copyTimestamps,
                 CancellationToken.None);
         }
+
+        /// <summary>
+        /// Gets a list of available fallback font files.
+        /// </summary>
+        /// <response code="200">Information retrieved.</response>
+        /// <returns>An array of <see cref="FontFile"/> with the available font files.</returns>
+        [HttpGet("FallbackFont/Fonts")]
+        [Authorize(Policy = Policies.DefaultAuthorization)]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        public IEnumerable<FontFile> GetFallbackFontList()
+        {
+            var encodingOptions = _serverConfigurationManager.GetEncodingOptions();
+            var fallbackFontPath = encodingOptions.FallbackFontPath;
+
+            if (!string.IsNullOrEmpty(fallbackFontPath))
+            {
+                var files = _fileSystem.GetFiles(fallbackFontPath, new[] { ".woff", ".woff2", ".ttf", ".otf" }, false, false);
+                var fontFiles = files
+                    .Select(i => new FontFile
+                    {
+                        Name = i.Name,
+                        Size = i.Length,
+                        DateCreated = _fileSystem.GetCreationTimeUtc(i),
+                        DateModified = _fileSystem.GetLastWriteTimeUtc(i)
+                    })
+                    .OrderBy(i => i.Size)
```

Comment:
More of a comment - if the goal is to return as many fonts to the user as possible - then wouldn't this be better being ordered smallest to largest?

Reply:
It is already arranged from small to large.

```
[{
	"Name": "NotoSansCJKkr-Regular.woff2",
	"Size": 1331160,
	"DateCreated": "2020-07-30T08:56:23.4010072Z",
	"DateModified": "2019-04-10T20:49:06Z"
}, {
	"Name": "NotoSansCJKsc-Regular.woff2",
	"Size": 1406148,
	"DateCreated": "2020-07-30T08:56:23.4331127Z",
	"DateModified": "2019-04-10T20:57:56Z"
}, {
	"Name": "NotoSansCJKjp-Regular.woff2",
	"Size": 1649860,
	"DateCreated": "2020-07-30T08:56:23.3598706Z",
	"DateModified": "2019-04-10T20:25:58Z"
}, {
	"Name": "NotoSansCJKtc-Regular.woff2",
	"Size": 3016668,
	"DateCreated": "2020-07-30T08:56:23.4923089Z",
	"DateModified": "2019-04-10T21:22:20Z"
}]
```

Reply:
Of course it is. Brain fart!